pkg/gpu/nvidia/nvml/nvml_manager.go

```diff
@@ -0,0 +1,48 @@
+package nvmlmanager
```

Comment:
Keeping with the go convention of package name being the last element of package path, can we name this `package nvml`?